kubectl/kubectl.bash

```diff
@@ -1,23 +1,32 @@
 #!/bin/bash
 
+versions=( $(cd "$(gcloud info --format='value(installation.sdk_root)')/bin/" && ls kubectl.* | sed s/kubectl.//g) )
+
 function var_usage() {
   cat <<EOF
   No cluster is set. To set the cluster (and the region/zone where it is found), set the environment variables
   CLOUDSDK_COMPUTE_REGION=<cluster region> (regional clusters)
   CLOUDSDK_COMPUTE_ZONE=<cluster zone>
   CLOUDSDK_CONTAINER_CLUSTER=<cluster name>
+
+  Optionally, you can specify the kubectl version via KUBECTL_VERSION, taking one of the following values: ${versions[@]}
 EOF
 
   exit 1
 }
 
+kubectl_cmd=kubectl.${KUBECTL_VERSION}
```

Comment:
This does not appear to be optional. If you do not specify `KUBECTL_VERSION`, it tries to run `kubectl.` and that doesn't exist. The period needs to only be added if `KUBECTL_VERSION` is set.

It appears this change has recently been pushed to `gcr.io` and is breaking our deployments. An urgent fix would be appreciated.

Reply:
e.g. `kubectl_cmd=kubectl${KUBECTL_VERSION:+.${KUBECTL_VERSION}}`

Reply:
oy. This is utter 🤦‍♂ and I'm so sorry. Let's get this fixed up ASAP